docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -130,4 +130,41 @@ When you attach an indicator to a case, the indicator is added as a new comment
 To remove an indicator attached to a case, click *More actions* (*...​*) -> *Delete attachment* in the case comment. 
 
 [role="screenshot"]
-image::images/remove-indicator.png[Removing an indicator from a case]
\ No newline at end of file
+image::images/remove-indicator.png[Removing an indicator from a case]
+
+[discrete]
+[[add-indicator-to-blocklists]]
+== Add indicators to blocklists
+
+Add indicators to blocklists to prevent selected applications from running on your hosts. You can add indicators of the `file` indicator type and that have hash values for the MD5, SHA-1, or SHA-256 fields.
+
+.Requirements
+[sidebar]
+--
+* In addition to configuring specific entries on the **Blocklist** page, you must also ensure that the blocklist is enabled on the {elastic-defend} integration policy in the <<malware-protection, Malware protection settings>>. This setting is enabled by default.
+
+* You must have the `superuser` {ref}/built-in-roles.html[built-in user role] to access this feature.
```

Comment:
This is actually changing in 8.7 due to [new RBAC for endpoint management](https://github.com/elastic/security-team/issues/5446). I haven't documented the RBAC features yet, but this probably works for now:

```suggestion
* You must have the `Blocklist` <<endpoint-management-req,feature privilege>> to access this feature.
```

We'll try to use a consistent format for privilege prereq statements like this, so if that changes once I document RBAC, I can update this PR as needed, or update the file itself if you've already merged this by that point.

Reply:
Good reminder, thanks. Feel free to update this line if I merge the PR before you get chance to doc the RBAC features.